client/extensions/woocommerce/app/dashboard/required-plugins-install-view.js

```diff
@@ -81,10 +89,10 @@ class RequiredPluginsInstallView extends Component {
 		for ( let i = 0; i < requiredPlugins.length; i++ ) {
 			const slug = requiredPlugins[ i ];
 			const plugin = find( plugins, { slug } );
+			if ( ! wporg[ slug ] ) {
+				return;
+			}
 			if ( ! plugin ) {
-				if ( ! wporg[ slug ] ) {
-					return;
-				}
 				const wporgPlugin = getPlugin( wporg, slug );
 				const progress = this.state.progress + ( 100 / requiredPlugins.length );
 				this.setState( {
```

Comment:
Should probably use callbacks on setState. There's been talk of deprecating the object implementation even.
https://facebook.github.io/react/docs/react-component.html#setstate